src/main/scala/io/kf/etl/ETLMain.scala

```diff
@@ -5,8 +5,13 @@ import io.kf.etl.processors.download.DownloadProcessor
 import io.kf.etl.processors.featurecentric.FeatureCentricProcessor
 import io.kf.etl.processors.index.IndexProcessor
 import io.kf.etl.processors.participantcommon.ParticipantCommonProcessor
+import io.kf.etl.processors.tojson.JsonOutputProcessor
+
+import scala.util.Try
 
 object ETLMain extends App {
+  val Array(saveJsonFiles) = args
+  val saveJsonToS3: Boolean = Try(saveJsonFiles.toBoolean).getOrElse(false)
```

Comment:
instead having a new parameter, could we relied on the same config that conatinas the filepath?